examples/hackernews/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to [Astro](https://astro.build) Hackernews
```

Comment:
```suggestion
# Astro Starter Kit: Hackernews

```
npm create astro@latest -- --template hackernews
```
```
 Just pointing out that there's another open PR #5201 that is standardizing some of the example READMEs.

Comparing to that, I think the only thing to keep this one in line with that model is updating the title to the  `Astro Starter Kit: Basics` with create astro command format.